broker/cloud_functions/ps_to_gcs/deploy.sh

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
+# Deploys or deletes broker Cloud Functions
+# This script will not delete Cloud Functions that are in production
+
+testid="${1:-test}"
+# "False" uses production resources
+# any other string will be appended to the names of all resources
+teardown="${2:-False}"
+# "True" tearsdown/deletes resources, else setup
+survey="${3:-ztf}"
+# name of the survey this broker instance will ingest
+versiontag="${4:-v3_3}"
+PROJECT_ID="${PROJECT_ID:-avid-heading-329016}"
```

Comment:
```suggestion
PROJECT_ID=$GOOGLE_CLOUD_PROJECT # get the environment variable
```

This needs to come from the environment variable `GOOGLE_CLOUD_PROJECT`, and it shouldn't get a default value.